qemu/tests/cfg/qemu_img_lock.cfg

```diff
@@ -0,0 +1,34 @@
+- qemu_img_lock:
+    only qcow2 raw
+    virt_test_type = qemu
+    force_create_image = no
+    start_vm = no
+    kill_vm = yes
+    backup_image_before_testing = yes
+    restore_image_after_testing = yes
+    store_vm_register = no
+    # use prefix ";" to separate the lock msg
+    lock_msg = 'Is another process using the image?'
+    variants:
+        - reject_boot_same_img_twice:
+            type = qemu_img_lock_reject_boot
+            second_vm_name = avocado-vt-vm2
+            lock_msg += ';Failed to get "write" lock'
+        - reject_boot_base_img_and_snapshot:
+            type = qemu_img_lock_reject_boot
+            create_snapshot = yes
+            not_preprocess = yes
+            force_create_image_sn = yes
+            images += " sn"
+            image_chain= "image1 sn"
+            image_name_sn = images/sn
+            image_format_sn = qcow2
+            second_vm_name = avocado-vt-vm2
+            lock_msg += ';Failed to get shared "write" lock'
+            variants:
+                - qcow2_format:
+                    only qcow2
+                    lock_msg += ';Could not open backing file:'
```

Comment:
Why are the lock messages different between qcow2 format and raw format? All of second boot would fail due the backing file is opened with a write lock. The original request for image locking is to make multiple QEMU processes cannot write to the same image. So it doesn't make much sense to check the detail message. And It may be changed in some day.

Reply:
Then checking the key word `"write" lock` would be enough. @hellenmao do you agree?

Reply:
That's okay to me.